README.md

```diff
@@ -23,13 +23,28 @@ be printed to your screen! Amazing, right?
 
 ## Use it!
 
+### As a base image:
+
 You can also use this image as a base image for your Carp-related Docker
 projects. This is as simple as starting your Dockerfile with:
 
 ```
 FROM carplang/carp:latest
 ```
 
+### As a carp builder:
+
+``` bash
+# from this repo root:
+docker run -v $(pwd)/example:/mnt/app \
+           --user 1000:1000 \
+           --rm \
+           carplang/carp:latest \
+           carp -b hello_world.carp
+
+./example/out/HelloWorld
```

Comment:
This will only work if the executable format is the same as in the image, so it won’t work on OS X for instance. Should we maybe clarify that?

Reply:
@hellerve  sure.

Reply:
@hellerve  done

Reply:
Thank you!